src/connectors/breadcrumb/__tests__/connectBreadcrumb-test.ts

```diff
@@ -173,12 +179,13 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/breadcrumb/
     it('returns the `SearchParameters` with the same `hierarchicalFacets` already defined', () => {
       const render = () => {};
       const makeWidget = connectBreadcrumb(render);
-      const helper = jsHelper({}, '', {
+      const helper = jsHelper(createSearchClient(), '', {
         hierarchicalFacets: [
           {
             name: 'category',
             attributes: ['category', 'sub_category'],
             separator: ' > ',
+            // @ts-ignore
             rootPath: null,
```

Comment:
Can this be changed to `undefined`?

Reply:
@francoischalifour Actually the `rootPath` do not exist on hierarchicalFacets type. 😢 

It's fine passing a ts ignore in this cases right? What do you propose?

Reply:
```
    type HierarchicalFacet = {
      name: string;
      attributes: string[];
      separator: string;
    };
```

Reply:
You mean that it's mistyped or that it's not supposed to be there?

Reply:
I am not sure what an `HierarchicalFacet` is. But at the moment, the algoliasearch-helper don't contain the property `rootPath` on the `HierarchicalFacet`. Therefore, if we don't ts ignore on tests, typescript complains that property don't exist in HierarchicalFacet.

Reply:
I meant, are the tests passing without `rootPath`? Perhaps it was not meant to be provided from the start.

Reply:
you  seem to have removed some of the other `rootPath: null`, but not this one. Is that on purpose?